types/index.d.ts

```diff
@@ -166,6 +166,7 @@ export namespace cart {
   function submitOrder(): Promise<Order>;
   function updateItem(id: string, input: CartItem): Promise<Cart>;
   function update(input: object): Promise<Cart>;
+  function getOrder(input?: string): Promise<Order>
```

Comment:
I might call `input` => `checkoutId`.

Reply:
Yes, I have updated those and I have also fixed the types for `ResultsResponse`, If I see any other missing type I will definitely open a pull request.